db/fixtures/ae_datastore/ManageIQ/System/Process.class/__methods__/event_router.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
@tinaafitz 
Can we call this parse_event_stream to be consistent with other methods